browser-interface/package.json

```diff
@@ -62,7 +62,7 @@
     "@dcl/hashing": "^1.1.3",
     "@dcl/kernel-interface": "^2.0.0-20230512115658.commit-b582e05",
     "@dcl/legacy-ecs": "^6.11.11",
-    "@dcl/protocol": "1.0.0-4982121558.commit-4b98e1a",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4982159613.commit-1ca0334.tgz",
```

Comment:
Once this PR is ready to merge, merge https://github.com/decentraland/protocol/pull/149 before to update this dependency with `npm i @dcl/protocol@next`